shared/runtime/sys_stdio_mphal.c

```diff
@@ -122,7 +122,7 @@ static const mp_stream_p_t stdio_obj_stream_p = {
 
 MP_DEFINE_CONST_OBJ_TYPE(
     stdio_obj_type,
-    MP_QSTR_FileIO,
+    MP_QSTR_StringIO,
```

Comment:
This should probably be `TextIOWrapper`, which matches CPython and also the unix port of MicroPython.

Reply:
Ah, even better.  I'll update and add some tests.